charts/hono/config/infinispan/hono-data-grid.xml

```diff
@@ -24,6 +24,16 @@
         <authorization enabled="true" />
       </security>
     </distributed-cache>
+    <distributed-cache name="command-router">
+      <encoding>
+        <key media-type="text/plain" />
+        <value media-type="text/plain" />
+      </encoding>
+      <memory storage="OFF_HEAP" max-size="30 MB" when-full="REMOVE"/>
+      <security>
+        <authorization enabled="true" />
+      </security>
+    </distributed-cache>
```

Comment:
why do we need to maintain two caches? Don't they serve the exact same purpose?

Reply:
They do server the same purpose (for now). Nonetheless, I would rather use the default cache names. Otherwise the cache name would have to be set explicitly in the config, which would make things rather confusing.

But I've amended the commit to now only have one "distributed-cache" block, with either the "command-router" name or "device-connection".